test/fail_compilation/fob1.d

```diff
@@ -13,7 +14,8 @@ fail_compilation/fob1.d(104): Error: variable `fob1.foo1.p` is returned but is U
     return p;
 }
 
-/* TEST_OUTPUT:
+/* REQUIRED_ARGS: -preview=dip1021
```

Comment:
The `REQUIRED_ARGS` should appear only once, at the start of the file, as all other occurrences are ignored (the file is compiled only once).